Servlet/Assignment 1/WebContent/WEB-INF/web.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://java.sun.com/xml/ns/j2ee" xmlns:web="http://xmlns.jcp.org/xml/ns/javaee"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-app_2_4.xsd http://xmlns.jcp.org/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
+	id="WebApp_ID" version="2.4">
+	<display-name>validation</display-name>
+	<servlet>
+		<display-name>doValidation</display-name>
+		<servlet-name>DoValidation</servlet-name>
+		<servlet-class>com.servlets.DoValidation</servlet-class>
+	</servlet>
+	<servlet-mapping>
```

Comment:
Keeping all the "<servlet-mapping>" tags in one place all together is a good practice.